CHANGELOG.md

```diff
@@ -25,11 +25,28 @@ New Features
 Bug Fixes
 ---------
 
+- Fixed a bug where a WebRTC PeerConnection was created before the application
+  had used the `connect()` API to join a Room. (JSDK-2392)
 - Fixed a bug where, the local and remote AudioTracks' audioLevels returned by
   `Room.getStats()` were not in the range [0-32767]. (JSDK-2303)
 - Fixed a bug where Chrome and Safari Participants were enabling simulcast for
   H264 LocalVideoTracks when VP8 simulcast was enabled. (JSDK-2321)
 
+Developer Notes
+---------------
+
+- On October 12, 2018, the specification for the JavaScript Session Establishment
+  Protocol (JSEP) was [updated](https://github.com/rtcweb-wg/jsep/pull/850) to say
+  that MediaStreamTrack IDs will no longer be present in Unified Plan SDPs
+  (Media Session Descriptions). twilio-video.js depends on them to map WebRTC
+  MediaStreamTracks to the appropriate RemoteAudioTracks and RemoteVideoTracks.
+  So, this release will make sure that MediaStreamTrack IDs are included in
+  Unified Plan SDPs. Once browsers implement this updated JSEP specification, you
+  will need to upgrade at least to this version of twilio-video.js for your application
+  to continue working. We will reach out to you with a detailed advisory once we
+  have more information from browser vendors about when they are planning to
+  implement the updated JSEP specification. (JSDK-2385)
+
```

Comment:
I tried to re-word hoping to clarify:

On October 12, 2018, the specification for the JavaScript Session Establishment Protocol (JSEP) was [updated](https://github.com/rtcweb-wg/jsep/pull/850) and got rid of MediaStreamTrack IDs from Unified Plan based SDPs (Media Session Descriptions). These updates impact twilio-video.js as it relies on the removed tack IDs to map WebRTC MediaStreamTracks to the corresponding RemoteAudioTracks and RemoteVideoTracks.
﻿﻿With this release of twilio-video.js, we have added support for ﻿﻿the updated JSEP specification. You will need to upgrade to this version so your application continues to function even when the browser vendors introduce support for the new JSEP specification.
We will provide a detailed advisory once we have more information from browser vendors about when they are planning to implement the updated JSEP specification. In the meantime, we recommend you upgrade to this version to avoid impact (JSDK-2385)